docs/guides/expectations/how_to_use_auto_initializing_expectations.md

```diff
@@ -33,7 +33,7 @@ To check whether the Expectation you are interested in works under the auto-init
 
 For example:
 
-```python title="Python script/Jupyter notebook" file=../../../tests/integration/docusaurus/expectations/auto_initializing_expectations/is_expectation_auto_initializing.py#L16-L18
+```python title="Python script/Jupyter notebook" name="tests/integration/docusaurus/expectations/auto_initializing_expectations/is_expectation_auto_initializing.py is_expectation_self_initializing False"
 ```
```

Comment:
I'm seeing some rendering issues with this one: 

This is the Netlify
![Screen Shot 2023-02-21 at 5 29 31 PM](https://user-images.githubusercontent.com/25670697/220497391-d230a1f1-add7-4335-acd5-7e195cde562b.png)


This is our docs site
![Screen Shot 2023-02-21 at 5 29 48 PM](https://user-images.githubusercontent.com/25670697/220497430-ffcddf2e-bbb0-4ac3-b4e3-0a3cdc8297bf.png)

Could you have a look?

Reply:
Good catch! Looking 👀

Reply:
That works! I am not sure of the root cause, but I think those titles are redundant so it is OK to just remove them. See here for new rendering: 
![image](https://user-images.githubusercontent.com/9903066/220645687-fcbf1049-e55c-420f-b388-43c0fcd13438.png)